gulpfile.js

```diff
@@ -0,0 +1,206 @@
+var autoprefixer = require('gulp-autoprefixer');
+var browserSync = require('browser-sync').create();
+var concat = require('gulp-concat');
+var csso = require('gulp-csso');
+var del = require('del');
+var eslint = require('gulp-eslint');
+var exec = require('gulp-exec');
+var gulp = require('gulp');
+var pjson = require('./package.json');
+var replace = require('gulp-replace');
+var sass = require('gulp-sass');
+var sourcemaps = require('gulp-sourcemaps');
+var stylelint = require('gulp-stylelint');
+var uglify = require('gulp-uglify');
+
+function cleanCSS() {
+  return del('dist/css');
+}
+
+function lintCSS() {
+  return gulp
+    .src(['src/scss/**/*.scss'])
+    .pipe(stylelint({
+      reporters: [
+        {
+          console: true,
+          formatter: 'string',
+        },
+      ],
+    }));
+}
+
+function buildCSS() {
+  return gulp
+    .src('src/scss/bootstrap-ui.scss')
+    .pipe(sourcemaps.init())
+    .pipe(sass())
+    .pipe(autoprefixer())
+    .pipe(concat('bootstrap-ui.css'))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('dist/css'));
+}
+
+function minifyCSS() {
+  return gulp
+    .src('dist/css/bootstrap-ui.css')
+    .pipe(sourcemaps.init())
+    .pipe(csso())
+    .pipe(concat('bootstrap-ui.min.css'))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('dist/css'))
+    .pipe(browserSync.stream());
+}
+
+function cleanJS() {
+  return del('dist/js');
+}
+
+function lintJS() {
+  return gulp
+    .src([
+      'src/js/*.js',
+      'src/js/tests/unit/*.js',
+      'gulpfile.js'
+    ])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failOnError());
+}
+
+function buildJS() {
+  return gulp
+    .src('src/js/*.js')
+    .pipe(sourcemaps.init())
+    .pipe(concat('bootstrap-ui.js'))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('dist/js'));
+}
+
+function minifyJS() {
+  return gulp
+    .src('dist/js/bootstrap-ui.js')
+    .pipe(sourcemaps.init())
+    .pipe(uglify())
+    .pipe(concat('bootstrap-ui.min.js'))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('dist/js'));
+}
+
+function cleanImages() {
+  return del('dist/images');
+}
+
+function copyImages() {
+  return gulp
+    .src('src/images/**')
+    .pipe(gulp.dest('dist/images'));
+}
+
+function bs(done) {
```

Comment:
two letter function names are not too descriptive. Could we write it out in full?